benchmarks/crossvalidation.py

```diff
@@ -0,0 +1,29 @@
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import GridSearchCV, cross_val_score
+
+from .common import Benchmark
+
+from .datasets import _synth_classification_dataset
+
+
+class CrossValidation_bench(Benchmark):
+    def setup(self, params):
+
+        self.n_jobs = params[0]
```

Comment:
it doesn't look clear for readers. why the first arg is n_jobs? if it is ASV format - ok, but if you can make it a dictionary and access by a key - it will be more clear